tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -115,11 +123,44 @@ def init_full_results(self, full_results):
         return full_results
 
     @pytest.mark.parametrize(
-        argnames=["pvc_size"],
+        argnames=["interface_type", "pvc_size"],
         argvalues=[
-            pytest.param(*["25Gi"], marks=pytest.mark.polarion_id("OCS-2007")),
-            pytest.param(*["50Gi"], marks=pytest.mark.polarion_id("OCS-2007")),
-            pytest.param(*["100Gi"], marks=pytest.mark.polarion_id("OCS-2007")),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, "5Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, "5Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL_THICK, "5Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, "15Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, "15Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL_THICK, "15Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, "25Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, "25Gi"],
+                marks=[pytest.mark.performance],
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL_THICK, "25Gi"],
+                marks=[pytest.mark.performance],
+            ),
```

Comment:
pleas change the order, all test from the same interface type together :
*[constants.CEPHBLOCKPOOL, "5Gi"]
*[constants.CEPHBLOCKPOOL, "15Gi"]
*[constants.CEPHBLOCKPOOL, "25Gi"]

*[constants.CEPHFILESYSTEM, "5Gi"]
*[constants.CEPHFILESYSTEM, "15Gi"]
*[constants.CEPHFILESYSTEM, "25Gi"]

*[constants.CEPHBLOCKPOOL_THICK, "5Gi"]
*[constants.CEPHBLOCKPOOL_THICK, "15Gi"]
*[constants.CEPHBLOCKPOOL_THICK, "25Gi"]

This is for the dashboard report.

Reply:
@Avilir done, thank you!